ksql-engine/src/main/java/io/confluent/ksql/function/udaf/topk/TopkKudaf.java

```diff
@@ -49,7 +49,8 @@
         argIndexInValue,
         ArrayList::new,
         returnType,
-        argumentTypes
+        argumentTypes,
+        "calculates the topk"
```

Comment:
It may make sense to spell out the semantic here, ie. it computes the topK values for the column, _per key_.